integration-tests/ci-visibility/test-api-manual/run-fake-test-framework.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable */
+require('./setup-fake-test-framework')
```

Comment:
```suggestion
'use strict'

/* eslint-disable */

require('./setup-fake-test-framework')
```

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/3202/commits/d7768fa1687dd12d86033ddc978e219c154fafca